connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractWorkerSourceTask.java

```diff
@@ -108,7 +108,7 @@ public abstract class AbstractWorkerSourceTask extends WorkerTask {
 
     /**
      * Invoked when a record is about to be dispatched to the producer. May be invoked multiple times for the same
-     * record if retriable errors are encountered.
+     * record if retryable errors are encountered.
```

Comment:
"Retriable" is correct and is used all over the code base ([here](https://github.com/apache/kafka/blob/4981fa939d588645401619bfc3e321dc523d10e7/clients/src/main/java/org/apache/kafka/common/errors/RetriableException.java) and [here](https://github.com/apache/kafka/blob/4981fa939d588645401619bfc3e321dc523d10e7/connect/api/src/main/java/org/apache/kafka/connect/errors/RetriableException.java), for example).

Reply:
got it, i got fooled by intellij :)